src/test/java/by/itechart/api/service/impl/UserServiceImplTest.java

```diff
@@ -85,8 +89,9 @@ void testDeleteExistingUser() {
     @DisplayName("Find all users")
     void testFindAllUsers() {
         List<User> suggestedAllUsers = List.of(new User(), new User());
-        when(userRepository.findAll()).thenReturn(suggestedAllUsers);
-        List<UserDTO> resultAllUsers = userService.findAll();
+        Page<User> pages = new PageImpl<>(suggestedAllUsers);
```

Comment:
Just nitpicking: misleading name
```suggestion
        Page<User> page = new PageImpl<>(suggestedAllUsers);
```